transfermethodui/src/main/res/values/strings.xml

```diff
@@ -57,20 +55,19 @@
     <string name="fee_mix_only_max_formatter">%1$s%2$s + %3$s%% (Max:%1$s %4$s)</string>
     <string name="fee_mix_formatter">%1$s%2$s + %3$s%% (Min:%1$s %4$s, Max:%1$s %5$s)</string>
 
-    <string name="account_holder">Account Holder</string>
-    <string name="account_information">Account Information</string>
-    <string name="address">Address</string>
-    <string name="contact_information">Contact Information</string>
-    <string name="identification">Identification</string>
-    <string name="intermediary_account">Intermediary Account</string>
+    <string name="account_holder">ACCOUNT HOLDER INFO</string>
+    <string name="account_information">ACCOUNT INFO</string>
+    <string name="address">ADDRESS</string>
+    <string name="contact_information">CONTACT INFO</string>
+    <string name="identification">IDENTIFICATION</string>
+    <string name="intermediary_account">INTERMEDIARY ACCOUNT</string>
+    <string name="mobileFeesAndProcessingTime">FEES AND PROCESSING TIME</string>
 
     <string name="search_hint">Search Hint</string>
     <string name="search_country">Search Country</string>
     <string name="search_currency">Search Currency</string>
     <string name="search_widget">Search</string>
 
-    <string name="account_information_section_header">Account Information - %s (%s)</string>
```

Comment:
As per invision: Account Info still needs to have the format `ACCOUNT INFO: COUNTRY(CURRENCY)`

Reply:
hmm I thought we will follow content, will confirm in. channel

Reply:
updated based on decision in channel